index.js

```diff
@@ -15,7 +15,6 @@
 var createError = require('http-errors')
 var debug = require('debug')('send')
 var deprecate = require('depd')('send')
-var destroy = require('destroy')
```

Comment:
@dougwilson so after you mentioned the `fd` leak I took a hard look at all the logic and it turned out you were right to bring it up. There were two situations where the `fd` would be leaked: 1) the `res` stream ends before it is passed to `sendStream.pipe()` and 2) the `res` ends after being passed to `sendStream.pipe()` but before `sendStream.send()` is called. I have accounted for these two situations in my latest commit and have added a few tests for them.